Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install |CHART_VERSION| --values values-migration.yaml --dry-run-helm-values > values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+When you install Cilium, it will create a new network interface called ``cilium_host`` on each node. You may need to configure the old CNI to ensure that it doesn't accidentally use this interface.
+
+For example, Calico has several modes of operation to decide which network interface to use by default on nodes that it manages. This is configured using the Tigera Operator's ``spec.calicoNetwork.nodeAddressAutodetectionV4`` (and respectively ``nodeAddressAutodetectionV6`` for IPv6) parameter.
+By default, the parameter is set to ``firstFound: true``, which uses the first detected network interface on the node. Check this value with:
+
+.. code-block:: shell-session
+
+      kubectl get installations.operator.tigera.io default \
+  -o jsonpath='{.spec.calicoNetwork.nodeAddressAutodetectionV4}{"\n"}'
+
+If Calico decides to use ``cilium_host`` as its default interface, Calico node routing will start failing. For this reason, we want to make sure that Calico ignores the ``cilium_host`` interface. 
+Depending on your Tigera Operator settings (for example, if you use the ``interface`` or ``skipInterface`` options), you can adjust the parameters to ensure ``cilium_host`` is not considered. 
+You can set ``firstFound`` to ``false`` and use ``kubernetes: NodeInternalIP`` instead, so Calico uses the node's internal IP as its main interface. 
```

Comment:
Grammar nits
```suggestion
If Calico decides to use ``cilium_host`` as its default interface, Calico node routing starts failing. To avoid failures, make sure that Calico ignores the ``cilium_host`` interface. 
Depending on your Tigera Operator settings (for example, if you use the ``interface`` or ``skipInterface`` options), you can adjust the parameters to ensure ``cilium_host`` is not considered. 
You can set ``firstFound`` to ``false`` and use ``kubernetes: NodeInternalIP`` instead, so that Calico uses the node's internal IP as its main interface. 
```